examples/react-native-query-suggestions/package.json

```diff
@@ -20,7 +20,7 @@
     "react-test-renderer": "16.8.6"
   },
   "dependencies": {
-    "algoliasearch": "3.35.1",
+    "algoliasearch": "4.0.0-beta.14",
```

Comment:
do we really want to ship examples with the beta version? I'm not sure it's fully stable, wdyt?

Reply:
To be honest `4.0.0-beta.14` is more stable than 3.35.1. 😄

Reply:
With stable I mean: few changes